x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/tests/pinned_events.ts

```diff
@@ -39,7 +38,7 @@ export default function ({ getService }: FtrProviderContextWithSpaces) {
     });
 
     describe('unpin an event', () => {
-      it('returns null', async () => {
+      it('returns { unpinned: true }', async () => {
         const response = await supertest
           .patch(PINNED_EVENT_URL)
```

Comment:
If this does become a generic kibana level feature, thinking this should be a delete instead of a patch...thoughts?

Reply:
Yes, for sure. The problem I was facing here was that it's not possible to define an empty and a non-empty API response for the same route in OpenAPI.